nengo/utils/functions.py

```diff
@@ -3,12 +3,17 @@
 from collections import OrderedDict
 
 import numpy as np
+import warnings
 
 from nengo.exceptions import ValidationError
 from nengo.utils.compat import is_number, iteritems
 
 
+
 def piecewise(data):
+    warnings.warn("piecewise() function is deprecated. Use Piecewise process instead.",
+                      DeprecationWarning)
```

Comment:
This line is overindented, according to PEP8. It should line up with the `p` in `piecewise`.